src/vmm/src/cpuid/cpuid_ffi.rs

```diff
@@ -532,54 +530,11 @@ impl fmt::LowerHex for RawKvmCpuidEntry {
 #[allow(clippy::unwrap_used, clippy::print_stdout, clippy::use_debug)]
 #[cfg(test)]
 mod tests {
-    #[cfg(cpuid)]
+
     use kvm_bindings::KVM_MAX_CPUID_ENTRIES;
 
+    use super::super::{CpuidRegisters, CpuidTrait};
     use super::*;
-    use crate::{CpuidRegisters, CpuidTrait};
-
-    #[test]
```

Comment:
are these tests gone now?

Reply:
I've removed these as they where broken by the change due to the path of the struct changing, they are brittle and I don't believe we  gain much value from testing `Debug` and `Display` implementations